tests/Oryx.BuildImage.Tests/CommandTests/PrepareEnvironmentCommandTest.cs

```diff
@@ -27,7 +27,7 @@ public PrepareEnvironmentCommandTest(ITestOutputHelper output) : base(output)
         {
         }
 
-        [Fact]
+        [Fact, Trait("category", "githubactions")]
```

Comment:
Some of these are listed as `github-actions`, some as `githubactions`. Based on the ci.yaml and nightly.yaml, as well as [the tests that actually ran](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=6330410&view=ms.vss-test-web.build-test-results-tab), I'm not sure that the `github-actions` tests are being included?

Reply:
As a side note, the test counts between the Before and After nightly runs (from in this ticket description) are pretty different. Do you know what is accounting for those differences in counts?

Reply:
Great catch! All GitHub Actions category should be `githubactions`. I've pushed a new change out